unit-testings/CMakeLists.txt

```diff
@@ -22,6 +23,9 @@ add_so_test(test_model-base-ekf-flex-estimator-imu)
 add_so_test(test-kinetics-observer)
 add_so_test(test-dcm-bias-estimation)
 add_so_test(test-kinematics)
+add_so_test(test-local-kinematics)
+#add_so_test(test-runge-kutta)
+#add_so_test(test-analytical-jacobians)
```

Comment:
Why are they not included?

Reply:
I now repaired test-analytical-jacobians and removed test-runge-kutta as the Runge-Kutta integration still needs some changes. About the Runge-Kutta integration: the analytical jacobians must be modified as the kinematics integration changed. Some other parts might also need corrections. The current state of the implementation of the Runge-Kutta integration has been switched to another branch: [fixRungeKutta](https://github.com/ArnaudDmt/state-observation/tree/fixRungeKutta).